en/controllers/components/authentication.rst

```diff
@@ -289,9 +289,13 @@ by including them in AuthComponents authenticate array::
 
     $this->Auth->config('authenticate', [
         'Openid', // app authentication object.
-        'AuthBag.Combo', // plugin authentication object.
+        'AuthBag.Openid', // plugin authentication object.
     ]);
 
+> Note that when using plugin notation there's no 'Authenticate' word when
```

Comment:
@elboletaire You'll find most information in the [guideline](http://book.cakephp.org/3.0/en/contributing/documentation.html)

basically here you could use `.. note::` and then indent the paragraph with 4 spaces.